tsconfig.cjs.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
can you please remove anything that's not a valid json? like every single comment in this file, thanks!

Reply:
Sure!